packages/experimental/AppearanceAdditions/ios/FRNAppearanceAdditions.m

```diff
@@ -67,9 +68,10 @@
 
 @implementation FRNAppearanceAdditions {
     BOOL _hasListeners;
-    NSString *_horizontalSizeClass;
-    NSString *_userInterfaceLevel;
-    NSString *_accessibilityContrastOption;
+    
+    NSMutableDictionary<NSNumber *, NSString *> * _rootTagHorizontalSizeClassMap;
```

Comment:
If you'd like to avoid `NSNumber` here (as per React's documentation), you could also use `id<NSCopying>`, which is sufficient to keep the dictionary happy.

Reply:
Also watch for spaces after the `*`, they should be right up against the variable name in our coding pattern.

Yes: `NSString *_string;`
No: `NSString * _string:`

Reply:
Fixed both! Thanks for the id<NSCopying> suggestion